src/components/Datatable/BatchModule/BatchActions/BatchActions.tsx

```diff
@@ -65,7 +67,7 @@ const BatchActions: React.FC<BatchActionsProps> = ({ actions }) => {
             (action as ActionWithSubactions<BatchActionArgs>).subActions,
           );
 
-          return (
+          element = (
```

Comment:
We have this conditional tooltip wrapper which could save us this reassigning; check the https://github.com/securityscorecard/design-system/blob/alpha/src/components/_internal/BaseTable/components/TooltipWrapper.tsx

Reply:
I noticed the tooltip component has the same functionality https://github.com/securityscorecard/design-system/blob/alpha/src/components/Tooltip/Tooltip.tsx#L20
The reassigning is because the component render different elements if it has subactions defined

Reply:
Right 👍